preheaders/checkout.php

```diff
@@ -450,11 +450,14 @@
 						$pmpro_msgt      = "pmpro_success";
 						$pmpro_confirmed = true;
 					} else {
+						// Track as spam activity to prevent too many CC attempts.

```

Comment:
Can we add a hook here instead perhaps, and have `pmpro_track_spam_activity` trigger on that hook?

Thinking about a hook here because it may be useful to allow people (or us) to implement other solutions to this problem via add ons.

Reply:
Yeah. I like pmpro_checkout_process_failed. Probably pass the order into it.